src/olympia/editors/helpers.py

```diff
@@ -53,8 +53,8 @@ def file_review_status(addon, file):
             return _(u'Pending Preliminary Review')
     # Special case: prelim upgrading to full approval,
     # file can already be preliminary reviewed or not
-    if (file.status in [amo.STATUS_LITE, amo.STATUS_UNREVIEWED]
-            and addon.status == amo.STATUS_LITE_AND_NOMINATED):
+    if (file.status in [amo.STATUS_LITE, amo.STATUS_UNREVIEWED] and
+            addon.status == amo.STATUS_LITE_AND_NOMINATED):
```

Comment:
file.status and addon.status
(I didn't change the logic, just moved the `and` to the line above to stop a pep8 complaint I was seeing)